build.sbt

```diff
@@ -293,6 +293,7 @@ def multiScalaDirectories(root: File, scalaVersion: String) = {
   if (isScala3(partialVersion)) {
     result += base / "scala-3"
   }
+  result += base / s"scala-$scalaVersion"
```

Comment:
There were issues in the build because mtags3 were lacking the version specific directory.

Reply:
Ironic I just hit on this when I cleaned and tried to build everything locally. It was on my list to look at after reviewing this, ha.